src/scripts/journal.js

```diff
@@ -6,7 +6,7 @@
 
 import API from "./data.js"
 import entryToDom from "./entriesDOM.js"
-import makeJournalEntryComponent from "./entryComponent.js"
+import { createJournalEntryHtml } from "./createJournalEntryHtml.js"
```

Comment:
The difference between importing a "named export" and a "default export" is the brackets... Named exports are imported just like how you saw the destructured arguments in the previous comment.